Pipfile

```diff
@@ -35,9 +35,11 @@ pycryptodomex = "*"
 
 [dev-packages]
 black = "==20.8b1"
-pytest = "*"
-pytest-cov = "*"
+mock = "*"
 mypy = "*"
+pytest = "*"
 pytest-asyncio = "*"
+pytest-cov = "*"
 requests = "*"
-mock = "*"
+types-mock = "*"
+types-protobuf = "*"
```

Comment:
`Pipfile` shouldn't be updated without an appropriate update of of the `Pipfile.lock`.